p2panda-rs/src/document/document.rs

```diff
@@ -573,7 +572,10 @@ mod tests {
             &update_operation(
                 schema.clone(),
                 vec![polar_entry_1_hash.clone().into()],
-                operation_fields(vec![("name", OperationValue::Text("🐼 Cafe!".to_string()))]),
+                operation_fields(vec![(
+                    "name",
+                    OperationValue::Text("🐼 Cafe!!".to_string()),
```

Comment:
I was able to retain the previous ordering of the document graph by adding a "!" :D

Reply:
Haha, brilliant 🤣